### PR TITLE
Copy edits for consistency

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
## Description

-  Fixes inconsistency in styling for doc

### Issue(s) fixed

Fixes #

### Preview

-  [Preview](https://besu-docs-git-minor-style-edits-hyperledger.vercel.app/development/public-networks/how-to/bonsai-limit-trie-logs)

### CODE REVIEW

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md [hunk 1]
```diff
@@ -125,9 +127,9 @@ Ensure you stop Besu before running the command.
 
 ###  Subcommand not working
```
Comment (m4sterbunny) [inline docs/public-networks/how-to/bonsai-limit-trie-logs.md:128]: drop H3s to H2s to follow H1

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md [hunk 2]
```diff
@@ -159,7 +161,7 @@ sudo /usr/local/bin/besu/bin/besu --data-path=/var/lib/besu --data-storage-forma
 
 ### Cannot store generated private key
 
-`java.lang.IllegalArgumentException: Cannot store generated private key`
+- `java.lang.IllegalArgumentException: Cannot store generated private key`
```
Comment (m4sterbunny) [inline docs/public-networks/how-to/bonsai-limit-trie-logs.md:164]: Recommend applying concatenate (we have mix of cannot and can't throughout these docs, probably need an issue just to slay)

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md [hunk 3]
```diff
@@ -170,7 +172,7 @@ sudo /usr/local/bin/besu/bin/besu --data-path=/var/lib/besu --data-storage-forma
 
 ### Valid keypair not provided
 
-`java.lang.IllegalArgumentException: Supplied file does not contain valid keyPair pair.`
+- `java.lang.IllegalArgumentException: Supplied file does not contain valid keyPair pair.`
```
Comment (m4sterbunny) [inline docs/public-networks/how-to/bonsai-limit-trie-logs.md:175]: ditto recommend concatenate does not > doesn't as per MS style

**Review by m4sterbunny — COMMENTED** [review]
inline comments

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md (hunk 2 above)
Comment (joaniefromtheblock) [reply docs/public-networks/how-to/bonsai-limit-trie-logs.md:164]: This is actually the error message in the terminal. It is taken from that message, so not a style guide edit ;/ but maybe we can raise it with the eng team.

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md (hunk 3 above)
Comment (joaniefromtheblock) [reply docs/public-networks/how-to/bonsai-limit-trie-logs.md:175]: same as above - in shell error message.

File: docs/public-networks/how-to/bonsai-limit-trie-logs.md (hunk 1 above)
Comment (joaniefromtheblock) [reply docs/public-networks/how-to/bonsai-limit-trie-logs.md:128]: These are under the troubleshoot section (H2), so should be under (H3)

Comment (joaniefromtheblock) [reply docs/public-networks/how-to/bonsai-limit-trie-logs.md:128]: These are under the troubleshoot section, so should be

**Review by m4sterbunny — APPROVED** [review]
LGTM

**Pull request merged**